bundles/org.openhab.binding.modbus.helioseasycontrols/README.md

```diff
@@ -277,6 +300,20 @@ The binding provides the following properties:
 | sensorConfigKwlFtf8              | Sensor configuration (installed or not) KWL-FTF 8 (OFF = no sensor, ON = sensor installed)                                       | 
 
 
+## Errors / Warnings / Infos
+
+Errors, warnings and infos of the device are provided in a bit encoded way. I.e. each bit in a 8 bit or 32 bit variable encodes potentially multiple eorrs, warnings or infos.
+Also status flags are provided this way. For details please refer to the manufacturer's [specification](https://www.easycontrols.net/de/service/downloads/send/4-software/16-modbus-dokumentation-f%C3%BCr-kwl-easycontrols-ger%C3%A4te).
+
+Based on that concept, errors, warnings and infos are provided 3 different ways:
+* As an integer value with the decimal representation of the encoded bits
```

Comment:
Unsigned integer? Or is it signed? Could be good to add

Reply:
It's unsigned. I've added this, makes certainly sense to be more precise here.